elements/pfe-dropdown/pfe-dropdown.ts

```diff
@@ -249,7 +265,11 @@ export class PfeDropdown extends LitElement {
   }
 
   /** modify DOM if custom options are passed in an array */
-  private async _modifyDOM(options: PfeDropdownOption[]) {
+  private async _modifyDOM(options: PfeDropdownOption[], clean: Boolean = true) {
+    // remove all dropdown items and separators
+    if (clean) {
+      this.innerHTML = '';
```

Comment:
seems expensive. might be better to iterate? we can defer that until we have numbers to back my assertion up

Reply:
I agree.  I always cringe with `innerHTML`.  I'll put in a `@todo` to revisit.